Assets/Scripts/Utilities/InputUtils.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public static class InputUtils
+{
+	// TODO ask Chris what this is even for
+	public static void CheckForController()
```

Comment:
This isn't necessary because you can setup the input in such a way that it handles mouse vs joystick correctly.